modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -48,21 +54,20 @@
 import java.util.Set;
 import java.util.TreeSet;
 
-
 public class ElmClientCodegen extends DefaultCodegen implements CodegenConfig {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DartClientCodegen.class);
```

Comment:
`ElmClientCodegen.class`

Reply:
Fixed. Sorry for the copy-and-paste error.